src/main/src/main/java/org/vfny/geoserver/global/CoverageInfoIndexComparator.java

```diff
@@ -0,0 +1,30 @@
+/* (c) 2014 Open Source Geospatial Foundation - all rights reserved
```

Comment:
New file, so there is no OpenPlans copyright, should only be "(c) 2019 Open Source Geospatial Foundation - all rights reserved"

Reply:
Also, why not have a generic ResourceInfoComparator instead? It would cover also WMS/WMTS cascaded layers and whatever else we might add in the future.